docs/standard/commandline/get-started-tutorial.md

```diff
@@ -0,0 +1,434 @@
+no-loc: [System.CommandLine]
+helpviewer_keywords:
+  - "command line interface"
+  - "command line"
+  - "System.CommandLine"
+---
+# Tutorial: Get started with System.CommandLine
+
+[!INCLUDE [scl-preview](../../../includes/scl-preview.md)]
+
+This tutorial shows how to create a .NET command-line app that uses the [`System.CommandLine` library](index.md). You'll begin by creating a simple root command that has one option. Then you'll add to that base, creating a more complex app that contains multiple subcommands and different options for each command.
+
+In this tutorial, you learn how to:
+
+> [!div class="checklist"]
+>
+> * Create commands, options, and arguments.
+> * Specify default values for options.
+> * Assign options and arguments to commands.
+> * Assign an option recursively to all subcommands under a command.
+> * Work with multiple levels of nested subcommands.
+> * Create aliases for commands and options.
+> * Work with `string`, `string[]`, `int`, `bool`, `FileInfo` and enum option types.
+> * Bind option values to command handler code.
+> * Use custom code for parsing and validating options.
+
+## Prerequisites
+
+* A code editor, such as [Visual Studio Code](https://code.visualstudio.com/) with the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp).
+* The [.NET 6 SDK](https://dotnet.microsoft.com/download/dotnet/6.0).
+
+The tutorial instructions show .NET CLI commands for project tasks, but you can use [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2022) with the **.NET desktop development** workload installed if you prefer.
+
+## Create the app
+
+Create a .NET 6 console app project named "scl".
+
+1. Create a folder named *scl* for the project, and then open a command prompt in the new folder.
+
+1. Run the following command:
+
+   ```dotnetcli
+   dotnet new console --framework net6.0
+   ```
+
+## Install the System.CommandLine package
+
+* Run the following command:
+
+  ```dotnetcli
+  dotnet add package System.CommandLine --prerelease
+  ```
+
+  The `--prerelease` option is necessary because the library is still in beta.
+
+1. Replace the contents of *Program.cs* with the following code:
+
+   :::code language="csharp" source="snippets/get-started-tutorial/csharp/Stage1/Program.cs" id="all" :::
+
+The preceding code:
+
+* Creates an [option](syntax.md#options) named `--file` of type `FileInfo` and assigns it to the [root command](syntax.md#commands):
+
+  :::code language="csharp" source="snippets/get-started-tutorial/csharp/Stage1/Program.cs" id="option" :::
+
+* Specifies that `ReadFile` is the method that will be called when the root command is invoked:
+
+  :::code language="csharp" source="snippets/get-started-tutorial/csharp/Stage1/Program.cs" id="sethandler" :::
+
+* Displays the contents of the specified file when the root command is invoked:
```

Comment:
Consider:

```suggestion
* The ReadFile method displays the contents of the specified file when the root command is invoked:
```

Reply:
The bullet points are continuations of the intro text: 

The preceding code... 

* Creates an option ...
* Specifies that ReadFile is the handler ... 
* Displays the file...